### DISCUSSION TITLE
Should Rolldown's Bench add `Bun.build` for another compare suite or not?

### DISCUSSION THREAD

**Discussion by Cecil0o0** [rolldown/rolldown#726] [General]
Bun team takes effort for realizing `Bun.build` API, they also aims for Speed, `Bun.build` seems to act as a bundler too, just like rolldown, I think. There If rolldown would add a new compare suite?

**Comment by hyfdev** [comment]
You could benchmark them in another repo. For reducing complexity, there's really no need to add many bundlers in this repo. It's enough so far to see if rolldown is fast just comparing with esbuild.

**Reply by hyfdev** [reply]
We might have a separate repo to benchmark with all bundlers in the future. But for the goal of seeing if rolldown is fast, the current benchmark in this repo is enough.
